### ISSUE TITLE
Compile (-c) error using query action

### ISSUE THREAD

**Issue by amreus** [red/red#3568]
Given this Red file `q.red`:

```red
Red [ file: "q.red" ]

foreach file read %. [
   q: query file
   ; print [file query file]
]
```

Generates the following error when using the command line: `red -c q.red` using `Red 0.6.3 for Windows built 16-Oct-2018/14:40:36-04:00 commit #7f4ed96` on Windows 10 Home x64.

```
-=== Red Compiler 0.6.3 ===- 

Compiling C:\Users\Amreus\Code\Red\red-lang-sandbox\utils\q.red ...
...using libRedRT built on 16-Oct-2018/22:23:02
...compilation time : 63 ms

Target: MSDOS 

Compiling to native code...
*** Compilation Error: undefined symbol: red/actions/query* 
*** in file: %/C/Users/Amreus/Code/Red/red-lang-sandbox/utils/q.red 
*** at line: 332 
*** near: [
    actions/query* 
    stack/unwind 
    word/set 
    stack/unwind
]
```

**Comment by dockimbel** [comment]
`query` action is not yet supported. It will be implemented in 0.7.0, with the rest of the missing I/O features.

**Comment by dockimbel** [comment]
I have cherry picked a [commit](https://github.com/red/red/commit/d11e5eb6cbda5cd83405011cbad1a60a9a326113) from `port-type` branch to enable libRedRT support for those I/O actions. So now your code should compile fine in dev mode.
